fabtests/test_configs/efa/efa-neuron.exclude

```diff
@@ -0,0 +1,94 @@
+#
```

Comment:
What is the use case for this file?

Reply:
Mainly to exclude the atomics tests as we're currently not supporting FI_HMEM and atomics with EFA due to bugs in those paths.